test/e2e/common/podtemplates.go

```diff
@@ -99,4 +100,79 @@ var _ = ginkgo.Describe("[sig-node] PodTemplates", func() {
 		framework.ExpectNoError(err, "failed to list PodTemplate")
 		framework.ExpectEqual(len(podTemplateList.Items), 0, "PodTemplate list returned items, failed to delete PodTemplate")
 	})
+
+	ginkgo.It("should delete a collection of pod templates", func() {
+		podTemplateNames := []string{"test-podtemplate-1", "test-podtemplate-2", "test-podtemplate-3"}
+		podTemplateNamesCount := len(podTemplateNames)
+
+		ginkgo.By("Create set of pod templates")
+		// create a set of pod templates in test namespace
+		for _, podTemplateName := range podTemplateNames {
+			_, err := f.ClientSet.CoreV1().PodTemplates(f.Namespace.Name).Create(context.TODO(), &v1.PodTemplate{
+				ObjectMeta: metav1.ObjectMeta{
+					Name:   podTemplateName,
+					Labels: map[string]string{"podtemplate-set": "true"},
+				},
+				Template: v1.PodTemplateSpec{
+					Spec: v1.PodSpec{
+						Containers: []v1.Container{
+							{Name: "nginx", Image: "nginx"},
```

Comment:
there is a agnhost image, and it has a pause command.
Let's update that here.

Reply:
[Fixed with](https://github.com/ii/kubernetes/commit/7772a42114f271296cd3817b58e5e025aa65905d#diff-a73abb634e908184c070916412d4e060R120) `Image: imageutils.GetE2EImage(imageutils.Agnhost)`